notes/interview.vbs

```diff
@@ -1194,6 +1194,11 @@ function define_main_dialog()
 			col_3_1 = 335
 			col_3_2 = 380
 
+			EditBox 225, y_pos -5, 145, 15, all_persons_paying
+			Text 	20, y_pos , 210, 10, "List household members who pay utilities (separated by commas):" 
+			y_pos = y_pos + 15
+
```

Comment:
I wonder if this should be a dropdown or checkbox of some sort. Since it is a free text field, the script will error if the incorrect HH memb numbers are used (such as putting in the names of the HH members instead of the number). Alternatively, maybe the instructions could be clearer that the HH Memb Number needs to be used - "List the household member numbers who pay utilities (separated by commas):"